### PR TITLE
chore(docs): refine style

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: apps/docs/assets/styles/main.css [hunk 1]
```diff
@@ -73,6 +87,60 @@ li .navigation {
...
+
+/*Main container*/
+/*--------------------------*/
+vwc-side-drawer::before {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 1px;
+  box-shadow: 0 1px 7px 0px var(--vvd-color-neutral-90);
+}
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:113]: not too fond of this one. it collides with the dynamic shadow's flow

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/assets/styles/main.css (hunk 1 above)
Comment (rachelbt) [reply apps/docs/assets/styles/main.css:113]: but theres no other way to add the shadow to the top app

Comment (yinonov) [reply apps/docs/assets/styles/main.css:113]: why add shadow?

Comment (rachelbt) [reply apps/docs/assets/styles/main.css]: I removed the shadow

**Review by yinonov — APPROVED** [review]

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -5,15 +5,20 @@ strong {
 /* #endregion reset */
 
 body {
-  color: var(--vvd-color-on-canvas);
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:8]: this breaks the style

Comment (yinonov) [reply apps/docs/assets/styles/main.css:8]: image

Comment (yinonov) [reply apps/docs/assets/styles/main.css:8]: typography and color are different concerns. text component should not control text color.
This must not merge as is

Comment (rachelbt) [reply apps/docs/assets/styles/main.css:8]: removed it after updating from main the change of the color-scheme

Comment (yinonov) [reply apps/docs/assets/styles/main.css:8]: the background and foreground will align to dark theme but wont be set as our tokens now

Comment (rachelbt) [reply apps/docs/assets/styles/main.css:8]: I added back the `bg color` and the `color` properties on `body`

**Review by yinonov — APPROVED** [review]

**Pull request merged**